### ISSUE TITLE
When trying to access apps.risevision.com it will sit on a loading spinner  

### ISSUE THREAD

**Issue by sheadarlison** [Rise-Vision/rise-vision-apps#626]
### What is the expected behavior?
When I go to apps.risevision.com and I'm logged in I see the page I want to access. 

### What is the actual behavior?
When I go to apps.risevision.com and I'm logged in I see a loading spinner and need to refresh the page multiple times to get to the page I want to access.

### What steps can be taken to replicate the problem?
I'm not sure, it happens to me almost every time I try to access apps and I'm logged in already. It's been happening for a while as well.

### Notes

1. MacOS High Sierra v10.13.4
1. Chrome Version 66.0.3359.139 (Official Build) (64-bit)
1. Video of the issue happening https://drive.google.com/file/d/1ZFZWkiKhM5FM1dgndvE9FnokU6IuIvP4/view

**Comment by ezequielc** [comment]
@sheadarlison Do you still have this issue? @alex-deaconu mentioned it happened to him as well, but he can no longer replicate. 
I am wondering if it was an intermittent issue that is no longer happening. Thanks

**Comment by sheadarlison** [comment]
@ezequielc I think it's intermittent, I'm only getting the spinner for about 4 seconds this morning but it was pretty bad yesterday afternoon.

**Comment by ezequielc** [comment]
@sheadarlison If you can get a screenshot of dev console, that would be helpful. 
It would be nice to check if there are network errors/request failures, but you will only see them if you keep the dev tools open before loading the page. Thanks!

**Comment by sheadarlison** [comment]
@ezequielc Okay I'll try to get some screenshots/video of those. Thanks

**Comment by sheadarlison** [comment]
@ezequielc It was happening pretty badly this morning. Here's the console errors. 

![screen shot 2018-05-16 at 10 29 32 am](https://user-images.githubusercontent.com/9273546/40123447-48cd1cf0-58f4-11e8-918c-16be70aef872.png)

![screen shot 2018-05-16 at 10 29 38 am](https://user-images.githubusercontent.com/9273546/40123461-4df8989e-58f4-11e8-8dd2-b328eec201ae.png)

![screen shot 2018-05-16 at 10 29 46 am](https://user-images.githubusercontent.com/9273546/40123468-513d0f6c-58f4-11e8-9ee7-4bfc5fa4f3e3.png)

**Comment by ezequielc** [comment]
Thanks @sheadarlison It will help a lot, we will take a look at it.

**Comment by alex-deaconu** [comment]
@sheadarlison from what I can tell in the screenshots, the page loads, but it takes a really long time. I just want to make sure it doesn't sit on the loading screen **forever** or until you refresh. Thanks.

**Comment by sheadarlison** [comment]
@alex-deaconu I've had it since, it's intermittent though, it'll happen like every few days. Thanks

**Comment by alex-deaconu** [comment]
@sheadarlison we have released a fix for this issue. To clarify, in your screenshot the Presentation API is returning a 503 error. This is an App Engine internal error which we cannot do much about. However, the fact that the app is stuck on the spinner is what we are addressing.

With this fix, it should no longer be stuck on the spinner but show a corresponding error for the list load.

Closing this issue. Let us know if this still occurs with the new version. Thanks!
